src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.Compute
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.Compute">
-      <PackageVersion>11.2.1</PackageVersion>
+      <PackageVersion>11.2.2</PackageVersion>
```

Comment:
It is a breaking change given the new data model contract changes in Replicate request payload and GetDetails response payload.